frontend/public/components/overview/deployment-overview.tsx

```diff
@@ -15,12 +14,23 @@ import {
 import { OverviewDetailsResourcesTab } from './resource-overview-page';
 import { OverviewItem } from '.';
 import { ResourceOverviewDetails } from './resource-overview-details';
+import PodRingDataContoller from './../../../packages/dev-console/src/components/topology/PodRingDataController';
+import PodsRing from './../../../packages/dev-console/src/components/PodsRing';
 
 const DeploymentOverviewDetails: React.SFC<DeploymentOverviewDetailsProps> = ({item}) => {
   return <div className="overview__sidebar-pane-body resource-overview__body">
     {item.obj.spec.paused && <WorkloadPausedAlert obj={item.obj} model={DeploymentModel} />}
     <div className="resource-overview__pod-counts">
-      <DeploymentPodCounts resource={item.obj} resourceKind={DeploymentModel} />
+      <PodRingDataContoller
+        namespace={item.obj.metadata.namespace}
+        knative={false}
+        render={({loaded, data: {pods: pods}}) => {
+          return <PodsRing pods={loaded ? pods[item.obj.metadata.uid].pods : []}
```

Comment:
While loading data, this will flash incorrect data stating that there are 0 pods.